### ISSUE TITLE
Bug: OneDrive folder is not correctly synchronized, only folders are created, files not downloaded

### ISSUE THREAD

**Issue by DavidGL17** [abraunegg/onedrive#2927]
### Describe the bug

On a brand new system, I want to sync my OneDrive folder locally. Running `onedrive --sync--verbose`, the folders are created but no file is downloaded. 

### Operating System Details

```shell
uname -a : Linux dgonzalezleon-ThinkPad-P14s-Gen-4 6.8.0-47-generic #47~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Oct  2 16:16:55 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

lsb_release -a : Ubuntu 22.04.5 LTS
```

### Client Installation Method

From Source

### OneDrive Account Type

Personal

### What is your OneDrive Application Version

onedrive v2.5.2-21-g197adad

### What is your OneDrive Application Configuration

```shell
Reading configuration file: /home/dgonzalezleon/.config/onedrive/config
Configuration file successfully loaded

WARNING: Your curl/libcurl version (7.81.0) has known HTTP/2 bugs that impact the use of this application.
         Please report this to your distribution and request that they provide a newer curl version for your platform or upgrade this yourself.
         Downgrading all application operations to use HTTP/1.1 to ensure maximum operational stability.
         Please read https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#compatibility-with-curl for more information.

Application version                          = onedrive v2.5.2-21-g197adad
Compiled with                                = DMD 2109
Curl version                                 = libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.3.1 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.18
User Application Config path                 = /home/dgonzalezleon/.config/onedrive
System Application Config path               = /etc/onedrive
Applicable Application 'config' location     = /home/dgonzalezleon/.config/onedrive/config
Configuration file found in config location  = true - using 'config' file values to override application defaults
Applicable 'sync_list' location              = /home/dgonzalezleon/.config/onedrive/sync_list
Applicable 'items.sqlite3' location          = /home/dgonzalezleon/.config/onedrive/items.sqlite3
Config option 'drive_id'                     = 
Config option 'sync_dir'                     = ~/OneDrive
Config option 'enable_logging'               = true
Config option 'log_dir'                      = ~/.config/onedrive
Config option 'disable_notifications'        = false
Config option 'skip_dir'                     = 
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp|*.swp|*.partial
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 12
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'disable_download_validation'  = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               = d50ca740-c83f-4d1b-b616-12c519384f0c
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = 
Config option 'user_agent'                   = ISV|abraunegg|OneDrive Client for Linux/v2.5.2-21-g197adad
Config option 'force_http_11'                = true
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'dns_timeout'                  = 60
Config option 'connect_timeout'              = 10
Config option 'data_timeout'                 = 60
Config option 'ip_protocol_version'          = 0
Config option 'threads'                      = 8
Config option 'max_curl_idle'                = 120
Compile time option --enable-notifications   = false

Selective sync 'sync_list' configured        = false

Config option 'sync_business_shared_items'   = false

Config option 'webhook_enabled'              = false
```

### What is your 'curl' version

```shell
curl 7.81.0 (x86_64-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.18
Release-Date: 2022-01-05
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets zstd
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
udev on /dev type devtmpfs (rw,nosuid,relatime,size=16133928k,nr_inodes=4033482,mode=755,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,noexec,relatime,size=3250152k,mode=755,inode64)
/dev/mapper/vgubuntu-root on / type ext4 (rw,relatime,errors=remount-ro)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k,inode64)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=29,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=22758)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
ramfs on /run/credentials/systemd-sysusers.service type ramfs (ro,nosuid,nodev,noexec,relatime,mode=700)
/var/lib/snapd/snaps/bare_5.snap on /snap/bare/5 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/airtame-application_25.snap on /snap/airtame-application/25 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/chromium_2972.snap on /snap/chromium/2972 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/chromium_2980.snap on /snap/chromium/2980 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core18_2829.snap on /snap/core18/2829 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core18_2846.snap on /snap/core18/2846 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core20_2379.snap on /snap/core20/2379 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core20_2434.snap on /snap/core20/2434 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core22_1612.snap on /snap/core22/1612 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core22_1621.snap on /snap/core22/1621 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core24_490.snap on /snap/core24/490 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/core24_609.snap on /snap/core24/609 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/cups_1058.snap on /snap/cups/1058 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/firefox_5014.snap on /snap/firefox/5014 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/firefox_5134.snap on /snap/firefox/5134 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-28-1804_198.snap on /snap/gnome-3-28-1804/198 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-38-2004_143.snap on /snap/gnome-3-38-2004/143 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gnome-42-2204_141.snap on /snap/gnome-42-2204/141 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gnome-42-2204_176.snap on /snap/gnome-42-2204/176 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gnome-46-2404_48.snap on /snap/gnome-46-2404/48 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/gtk-common-themes_1535.snap on /snap/gtk-common-themes/1535 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/kubectl_3419.snap on /snap/kubectl/3419 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/localsend_18.snap on /snap/localsend/18 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/mesa-2404_143.snap on /snap/mesa-2404/143 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/notion-snap-reborn_46.snap on /snap/notion-snap-reborn/46 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/notion-snap-reborn_47.snap on /snap/notion-snap-reborn/47 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snap-store_1113.snap on /snap/snap-store/1113 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snap-store_1216.snap on /snap/snap-store/1216 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snapd_21759.snap on /snap/snapd/21759 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snapd-desktop-integration_247.snap on /snap/snapd-desktop-integration/247 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snapd-desktop-integration_253.snap on /snap/snapd-desktop-integration/253 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/snapd_20671.snap on /snap/snapd/20671 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/spotify_78.snap on /snap/spotify/78 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/spotify_79.snap on /snap/spotify/79 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/teams-for-linux_689.snap on /snap/teams-for-linux/689 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/dev/mapper/vgubuntu-root on /var/snap/firefox/common/host-hunspell type ext4 (ro,noexec,noatime,errors=remount-ro)
/var/lib/snapd/snaps/telegram-desktop_6238.snap on /snap/telegram-desktop/6238 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/telegram-desktop_6246.snap on /snap/telegram-desktop/6246 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/var/lib/snapd/snaps/teams-for-linux_691.snap on /snap/teams-for-linux/691 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
/dev/nvme0n1p2 on /boot type ext4 (rw,relatime)
/dev/nvme0n1p1 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
tmpfs on /run/snapd/ns type tmpfs (rw,nosuid,nodev,noexec,relatime,size=3250152k,mode=755,inode64)
nsfs on /run/snapd/ns/cups.mnt type nsfs (rw)
nsfs on /run/snapd/ns/snapd-desktop-integration.mnt type nsfs (rw)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,size=3250148k,nr_inodes=812537,mode=700,uid=1000,gid=1000,inode64)
portal on /run/user/1000/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
nsfs on /run/snapd/ns/snap-store.mnt type nsfs (rw)
nsfs on /run/snapd/ns/telegram-desktop.mnt type nsfs (rw)
nsfs on /run/snapd/ns/spotify.mnt type nsfs (rw)
nsfs on /run/snapd/ns/firefox.mnt type nsfs (rw)
/var/lib/snapd/snaps/kubectl_3446.snap on /snap/kubectl/3446 type squashfs (ro,nodev,relatime,errors=continue,threads=single,x-gdu.hide)
nsfs on /run/snapd/ns/teams-for-linux.mnt type nsfs (rw)
nsfs on /run/snapd/ns/notion-snap-reborn.mnt type nsfs (rw)
overlay on /var/lib/docker/overlay2/84ddf982de6fe589e4e5ac07f2a007eac66df74355ad26109a2cc0d8fe63b97e/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/IFV3G4IMI5G5Q4XPVVP5JYUNY4:/var/lib/docker/overlay2/l/WYBVPWY4OQBDHOPQYCJ3TTH7TG,upperdir=/var/lib/docker/overlay2/84ddf982de6fe589e4e5ac07f2a007eac66df74355ad26109a2cc0d8fe63b97e/diff,workdir=/var/lib/docker/overlay2/84ddf982de6fe589e4e5ac07f2a007eac66df74355ad26109a2cc0d8fe63b97e/work,nouserxattr)
overlay on /var/lib/docker/overlay2/aef16af3c6e6f8e42e481b34a52111ecbbacf5ac542a1ed24f85cad0625d509c/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/RQYG3YELY6KHNEXLVMD4XTD6V7:/var/lib/docker/overlay2/l/6O65IOLUUQPUAGAOD646VWFRBT,upperdir=/var/lib/docker/overlay2/aef16af3c6e6f8e42e481b34a52111ecbbacf5ac542a1ed24f85cad0625d509c/diff,workdir=/var/lib/docker/overlay2/aef16af3c6e6f8e42e481b34a52111ecbbacf5ac542a1ed24f85cad0625d509c/work,nouserxattr)
overlay on /var/lib/docker/overlay2/e3dd80bc4c89defbf772d74af6a6b0d33b16398b29467e660588bcf188bc2d7a/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/IHLAHNTU5UIARSRHU7RQA7FT7V:/var/lib/docker/overlay2/l/RYTFKAHYGA7ZME4XVPFXACQUHY:/var/lib/docker/overlay2/l/YD5KPZTKIMO6Y77IFADMHWWTWK:/var/lib/docker/overlay2/l/ZG3SDQ5WMAVRT6RB3QKYYPREPU:/var/lib/docker/overlay2/l/W6FVGIWK4LK2GA6HHFPOYAK36J:/var/lib/docker/overlay2/l/2DFOWMXKSDJAFR2VWI2G25JK4U:/var/lib/docker/overlay2/l/RTI2XOAW36YSYTEGRXJUR75XOS,upperdir=/var/lib/docker/overlay2/e3dd80bc4c89defbf772d74af6a6b0d33b16398b29467e660588bcf188bc2d7a/diff,workdir=/var/lib/docker/overlay2/e3dd80bc4c89defbf772d74af6a6b0d33b16398b29467e660588bcf188bc2d7a/work,nouserxattr)
overlay on /var/lib/docker/overlay2/a809ec48071162d34374f774560ccdc7b934db010b04408220e61e25906f52e1/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/4GYA5PFD7UOLTGF7WL6KN4JVEA:/var/lib/docker/overlay2/l/O4ZHKJLQZNVEZMXVXDZPNQ4QXP:/var/lib/docker/overlay2/l/KJDK5JYIWKPBVK53PMFFQUAFG2:/var/lib/docker/overlay2/l/HGCHGN4ATWGFOLFXYVX6RLYRDY:/var/lib/docker/overlay2/l/EXTQDJUZ5YPSOK5Q2LNAINKAOJ:/var/lib/docker/overlay2/l/XTBVP6RY2YKQR77YKTMXCVGCC4:/var/lib/docker/overlay2/l/NK2LQXD2L7W2UMGJD5IYSBGTB4:/var/lib/docker/overlay2/l/EPG52FEIJYX7KXLFMCPJ6F6A2W,upperdir=/var/lib/docker/overlay2/a809ec48071162d34374f774560ccdc7b934db010b04408220e61e25906f52e1/diff,workdir=/var/lib/docker/overlay2/a809ec48071162d34374f774560ccdc7b934db010b04408220e61e25906f52e1/work,nouserxattr)
overlay on /var/lib/docker/overlay2/f6fd53f55a374f7beddd533f822d2bffaea1ca5fa4de3a7dfa5bfa25a5d83afb/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/EQMLM4SUUHTLZOSNM2EOWOG3TD:/var/lib/docker/overlay2/l/5RTXUCRMDPZUZMGR5Z5ZMJ5LNQ:/var/lib/docker/overlay2/l/QTU6I5V7K2IUFNX7QILB2TCH5H:/var/lib/docker/overlay2/l/PDVVSCWKFGBNIYIADBC27R6JAO:/var/lib/docker/overlay2/l/OY77BK6VVFFKFGTBKJKQGPVARZ:/var/lib/docker/overlay2/l/7NRQ2S7TBS2O6SCJDJG4D2GVYP:/var/lib/docker/overlay2/l/KYZ5ZYBRO236CQIXG7STCAWNJM:/var/lib/docker/overlay2/l/K6HJEVVKYF4W37GVJIMI4WYIXM:/var/lib/docker/overlay2/l/I3EQ4U5VREA4CGHL3JC3OXRIH2:/var/lib/docker/overlay2/l/TECV7FQL3HKBU24S7HZ3TTCGFA:/var/lib/docker/overlay2/l/TLWQARASD7DQZX2LKRRHIXS36I:/var/lib/docker/overlay2/l/UXLPJDBCN67UWEOZ2FHTFCMOM6:/var/lib/docker/overlay2/l/X5Y2NTELAXQ2QFHQNMQY2XT75W:/var/lib/docker/overlay2/l/L6COJ3BUR2JO5AYEMC3BGNEI27:/var/lib/docker/overlay2/l/B5AYV2E5SX7C3CW5FBXVMLWTW4:/var/lib/docker/overlay2/l/6753I4F77QT2R4G5JAKPJYO4TV:/var/lib/docker/overlay2/l/PUNOUZSCGNDXUC3LCPFUCH6UER:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/f6fd53f55a374f7beddd533f822d2bffaea1ca5fa4de3a7dfa5bfa25a5d83afb/diff,workdir=/var/lib/docker/overlay2/f6fd53f55a374f7beddd533f822d2bffaea1ca5fa4de3a7dfa5bfa25a5d83afb/work,nouserxattr)
overlay on /var/lib/docker/overlay2/a28b8559a8439e53f3429dd1fa4e9574aab78e93447a1952ff7ba429f9116e9e/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/P6VKVXOKJFCIVWIDZZUDETAJMN:/var/lib/docker/overlay2/l/XHSEL3QU7ZK75TL3KKLDORF3QI:/var/lib/docker/overlay2/l/YJ7HO5KGK7XBWLTTKXK3RZHEIM:/var/lib/docker/overlay2/l/WHVSYV4UVZZBSR3WS2HMGME76Y:/var/lib/docker/overlay2/l/6WPPUTMHWFYXPO65DZZQAVKQ7A:/var/lib/docker/overlay2/l/YUAXICZKWIHPF3ZH7533O772MC:/var/lib/docker/overlay2/l/JMBVXCZRBHQ4Z7PDAMAII2KNA5:/var/lib/docker/overlay2/l/MPFIQYWQ4CH7DZK6N74PTZEOA7:/var/lib/docker/overlay2/l/PSWQV37WXGWZ5MJ6HIBP24OHAE:/var/lib/docker/overlay2/l/JWJXUYLBD5WEVGHCLFC6WHAIPO:/var/lib/docker/overlay2/l/E37L4WXFMOZHXKL7UI4HD5X3T3,upperdir=/var/lib/docker/overlay2/a28b8559a8439e53f3429dd1fa4e9574aab78e93447a1952ff7ba429f9116e9e/diff,workdir=/var/lib/docker/overlay2/a28b8559a8439e53f3429dd1fa4e9574aab78e93447a1952ff7ba429f9116e9e/work,nouserxattr)
overlay on /var/lib/docker/overlay2/94a5bd108ecd58a1b5d5e05c5e0c6a959d40274ffaa98bb14a27d921e36e3ee7/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/R7VT2OO7YBWW7PO3U76WMKHVS7:/var/lib/docker/overlay2/l/CW55HDE35I6XDAXVESLFSLPGMJ:/var/lib/docker/overlay2/l/PD7A54WH6CZXHCPWRPHQWCHHIY:/var/lib/docker/overlay2/l/EIUJDU3VO3EWHGTHSB3VIDYEW2:/var/lib/docker/overlay2/l/54AZHHBMSPBKSP43TNAABZVC42:/var/lib/docker/overlay2/l/GO3JFDTWRMHYF37TONEHQPEWL7:/var/lib/docker/overlay2/l/OT26VI2Y4LRWEOUJU7C5ZLRIAQ:/var/lib/docker/overlay2/l/HQO4VFTBWZK3IM3EH23IRY6QFU:/var/lib/docker/overlay2/l/QVTTB3N4RJXEXAFQUH7DOXQUEN:/var/lib/docker/overlay2/l/VDWD6KASWBRFTNGUFX7FNNMJ54:/var/lib/docker/overlay2/l/T54NB5MHZRDZLTPE3LX67LPKV2:/var/lib/docker/overlay2/l/C3C23FR4LLDET7UKJZUX23XFAZ:/var/lib/docker/overlay2/l/7WZTPPNKQAYGK2IFSYQL3OXOG4:/var/lib/docker/overlay2/l/CJS3UG3IDZUUMZMF3OY6KOHBYP:/var/lib/docker/overlay2/l/VNUYQWUWERNNOAGVTJNWCXIZQD,upperdir=/var/lib/docker/overlay2/94a5bd108ecd58a1b5d5e05c5e0c6a959d40274ffaa98bb14a27d921e36e3ee7/diff,workdir=/var/lib/docker/overlay2/94a5bd108ecd58a1b5d5e05c5e0c6a959d40274ffaa98bb14a27d921e36e3ee7/work,nouserxattr)
overlay on /var/lib/docker/overlay2/94fef614f019555f8f9f456cd60e98034038ec7bb60fe2748d648d42edf6c99a/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/L4OPTJTWY3UNZJJJ6CDNP2M4DQ:/var/lib/docker/overlay2/l/N7SNZILV2MCYPVWCNDD57LVL44:/var/lib/docker/overlay2/l/UX3B4DGKJR6ECWIZHWEMIE3JXC:/var/lib/docker/overlay2/l/AE54TFI6AYMXEAV4U4IYRXYGGV:/var/lib/docker/overlay2/l/OGZIPERM6M27YA4B3FKMK5UBQU:/var/lib/docker/overlay2/l/X3TQNOVDVN5GBKT65QFQU2KNFB:/var/lib/docker/overlay2/l/KEGGQNZ4BKOAYO76X7PZSW3TG6:/var/lib/docker/overlay2/l/GOLM6XU3Y43XGMEKQ46IKBOB2P:/var/lib/docker/overlay2/l/GXKTA3AC4NYR3T3DAPWKX2DYFA:/var/lib/docker/overlay2/l/OBKPH2YOG2Z5FDEX5HV2FSMUHR:/var/lib/docker/overlay2/l/6342AJW5YGUWLZ6ZLBTLUSHGC6:/var/lib/docker/overlay2/l/4V5TNMXZAPW4TJ5NPD7BJXDZHV:/var/lib/docker/overlay2/l/G726LFKNFZPQE5NPVL4DNTL6LX:/var/lib/docker/overlay2/l/MDFW6YF3B4WILJQBKQNQ7ESIVV:/var/lib/docker/overlay2/l/MLWPB7FDQ2GXCUFRAFYHKSKIDW:/var/lib/docker/overlay2/l/43MMVUZRQ3JYVB72U4PKJM3XBA:/var/lib/docker/overlay2/l/UWYADRELCZC2OAR66AWRNMXRPS:/var/lib/docker/overlay2/l/5MNTMBPFXF6QSDKW6DMTP7QT6B:/var/lib/docker/overlay2/l/RR3WG2I3YPKWZJB25R2U45A6PI:/var/lib/docker/overlay2/l/TLWQARASD7DQZX2LKRRHIXS36I:/var/lib/docker/overlay2/l/UXLPJDBCN67UWEOZ2FHTFCMOM6:/var/lib/docker/overlay2/l/X5Y2NTELAXQ2QFHQNMQY2XT75W:/var/lib/docker/overlay2/l/L6COJ3BUR2JO5AYEMC3BGNEI27:/var/lib/docker/overlay2/l/B5AYV2E5SX7C3CW5FBXVMLWTW4:/var/lib/docker/overlay2/l/6753I4F77QT2R4G5JAKPJYO4TV:/var/lib/docker/overlay2/l/PUNOUZSCGNDXUC3LCPFUCH6UER:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/94fef614f019555f8f9f456cd60e98034038ec7bb60fe2748d648d42edf6c99a/diff,workdir=/var/lib/docker/overlay2/94fef614f019555f8f9f456cd60e98034038ec7bb60fe2748d648d42edf6c99a/work,nouserxattr)
overlay on /var/lib/docker/overlay2/273877d84dab5c9b12773af8b39e6ae25ec3142ef64fd0ee51fc8ac270e20a0a/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/ADQ5NU73WYQG5Z3IOMTC2SUFWX:/var/lib/docker/overlay2/l/FDPVZPQFH3VZJTOSE753ELW5GV:/var/lib/docker/overlay2/l/JSOH3UUHLDYFLTKJ3YGSX3JJ5P:/var/lib/docker/overlay2/l/DZZVYCC2P4MHVLSE3TLRF3KKQV:/var/lib/docker/overlay2/l/2HHR5NCSSNXA4PHHTZR5N3R2JL:/var/lib/docker/overlay2/l/LEMR5OB5TAEEJN2EO6NWRTHPC4:/var/lib/docker/overlay2/l/S57F3RTZMGI46YPZ6KOMNSCCDY:/var/lib/docker/overlay2/l/QQEMNM3A4EUBDD7GSHAK3DQATU:/var/lib/docker/overlay2/l/BBORY7Y6MNZMSZG2D4ABYRP23L:/var/lib/docker/overlay2/l/O5BOIIXGASSGZLN6EJUI73HVY5,upperdir=/var/lib/docker/overlay2/273877d84dab5c9b12773af8b39e6ae25ec3142ef64fd0ee51fc8ac270e20a0a/diff,workdir=/var/lib/docker/overlay2/273877d84dab5c9b12773af8b39e6ae25ec3142ef64fd0ee51fc8ac270e20a0a/work,nouserxattr)
nsfs on /run/docker/netns/c948bd58935a type nsfs (rw)
nsfs on /run/docker/netns/7ceb4515f3c8 type nsfs (rw)
nsfs on /run/docker/netns/0ac67cb6a1df type nsfs (rw)
nsfs on /run/docker/netns/ca059edf564d type nsfs (rw)
nsfs on /run/docker/netns/6bf20ce645d4 type nsfs (rw)
nsfs on /run/docker/netns/bf99e0b48a7d type nsfs (rw)
nsfs on /run/docker/netns/897ab0e6bbd4 type nsfs (rw)
nsfs on /run/docker/netns/56d4ff39c317 type nsfs (rw)
nsfs on /run/docker/netns/739ec0e39e2f type nsfs (rw)
overlay on /var/lib/docker/overlay2/49eabf75309ec9a5c6536f513a20d4699bc2e61da11e77da8c931f35bf6ac307/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/VZV23FGZCVE54ZGI7MAFRAGWDR:/var/lib/docker/overlay2/l/5RTXUCRMDPZUZMGR5Z5ZMJ5LNQ:/var/lib/docker/overlay2/l/QTU6I5V7K2IUFNX7QILB2TCH5H:/var/lib/docker/overlay2/l/PDVVSCWKFGBNIYIADBC27R6JAO:/var/lib/docker/overlay2/l/OY77BK6VVFFKFGTBKJKQGPVARZ:/var/lib/docker/overlay2/l/7NRQ2S7TBS2O6SCJDJG4D2GVYP:/var/lib/docker/overlay2/l/KYZ5ZYBRO236CQIXG7STCAWNJM:/var/lib/docker/overlay2/l/K6HJEVVKYF4W37GVJIMI4WYIXM:/var/lib/docker/overlay2/l/I3EQ4U5VREA4CGHL3JC3OXRIH2:/var/lib/docker/overlay2/l/TECV7FQL3HKBU24S7HZ3TTCGFA:/var/lib/docker/overlay2/l/TLWQARASD7DQZX2LKRRHIXS36I:/var/lib/docker/overlay2/l/UXLPJDBCN67UWEOZ2FHTFCMOM6:/var/lib/docker/overlay2/l/X5Y2NTELAXQ2QFHQNMQY2XT75W:/var/lib/docker/overlay2/l/L6COJ3BUR2JO5AYEMC3BGNEI27:/var/lib/docker/overlay2/l/B5AYV2E5SX7C3CW5FBXVMLWTW4:/var/lib/docker/overlay2/l/6753I4F77QT2R4G5JAKPJYO4TV:/var/lib/docker/overlay2/l/PUNOUZSCGNDXUC3LCPFUCH6UER:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/49eabf75309ec9a5c6536f513a20d4699bc2e61da11e77da8c931f35bf6ac307/diff,workdir=/var/lib/docker/overlay2/49eabf75309ec9a5c6536f513a20d4699bc2e61da11e77da8c931f35bf6ac307/work,nouserxattr)
nsfs on /run/docker/netns/8c621f5c4e05 type nsfs (rw)
overlay on /var/lib/docker/overlay2/4c99e8e7c27f38f27f4181604bc00e8408e59f8653641635dbb253610e6866fe/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/XABVKKXVM62JZCODJYBZU5NGVU:/var/lib/docker/overlay2/l/A5OG5VAWRL3H2VWRE36PHEKK32:/var/lib/docker/overlay2/l/6RWABRG64SJGJHIAVVNAAJGFQ6:/var/lib/docker/overlay2/l/GGI6GPP3PR474PKB2LGKIZRX3W:/var/lib/docker/overlay2/l/4YOSQ5O5L4IMZS3PHRFBVNNCYE:/var/lib/docker/overlay2/l/NV2AI2UF2RAXC7LCZDCN7EEHC3:/var/lib/docker/overlay2/l/2EJTWTFJ2E4AFFDBHNWO36DKOB:/var/lib/docker/overlay2/l/ECWUI3ZYLGQLBEXJ4BMQRAQIAS:/var/lib/docker/overlay2/l/O4ZHKJLQZNVEZMXVXDZPNQ4QXP:/var/lib/docker/overlay2/l/KJDK5JYIWKPBVK53PMFFQUAFG2:/var/lib/docker/overlay2/l/HGCHGN4ATWGFOLFXYVX6RLYRDY:/var/lib/docker/overlay2/l/EXTQDJUZ5YPSOK5Q2LNAINKAOJ:/var/lib/docker/overlay2/l/XTBVP6RY2YKQR77YKTMXCVGCC4:/var/lib/docker/overlay2/l/NK2LQXD2L7W2UMGJD5IYSBGTB4:/var/lib/docker/overlay2/l/EPG52FEIJYX7KXLFMCPJ6F6A2W,upperdir=/var/lib/docker/overlay2/4c99e8e7c27f38f27f4181604bc00e8408e59f8653641635dbb253610e6866fe/diff,workdir=/var/lib/docker/overlay2/4c99e8e7c27f38f27f4181604bc00e8408e59f8653641635dbb253610e6866fe/work,nouserxattr)
nsfs on /run/docker/netns/d8affab66fe5 type nsfs (rw)
overlay on /var/lib/docker/overlay2/a4a62f0d5d14ed7b08c27c1980eccbd721a133450e2c37b241bd3c43ab94e99b/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/7DHHWXBXUJYIH3544LIHDSZNRU:/var/lib/docker/overlay2/l/HJF6NFFA3BUYNDVWR65ETDEXBK:/var/lib/docker/overlay2/l/BEL5MD4RZRNLOLYUWW25TFPA5J:/var/lib/docker/overlay2/l/ISNPELTCLCBHUXSBMRDPEXNG4O:/var/lib/docker/overlay2/l/IGEYAOIFHTZEE2S62HNOJH7DWC:/var/lib/docker/overlay2/l/CDG5QDZ3KXLAA2D3EO3OIV2PJW:/var/lib/docker/overlay2/l/GJKFB55PZLIYWP6XHKBUBOL4RY:/var/lib/docker/overlay2/l/PHL2PB32SAPJV2K4CQ5HE7DOWD:/var/lib/docker/overlay2/l/EQ7QH4EHRGQWA4EU4XNNR2MLA4:/var/lib/docker/overlay2/l/6NFMXFBHCB5A4XFXTVSCLAXPHQ:/var/lib/docker/overlay2/l/QJEO62WLEMWTU4DUDTOX6JGWPW:/var/lib/docker/overlay2/l/ZZBZWL7T2ITEVJZV7QEHXUTL65:/var/lib/docker/overlay2/l/TV75BFF5MCZWYETCW4FRDBOGDP:/var/lib/docker/overlay2/l/LS2EPQCRL3DW7TZL4KMRIKZQIA:/var/lib/docker/overlay2/l/2S37QQXPP2HISKIDOBXNDKC542:/var/lib/docker/overlay2/l/P6EK4U4AYOBL4CVCQMHCRXICR7:/var/lib/docker/overlay2/l/WYMMU36XD2N5P7XODXRUQ7GWGP:/var/lib/docker/overlay2/l/IIDNLR4NH2W3RHNSFUIH5BAE7H:/var/lib/docker/overlay2/l/TYJ5ZK2NV3TRNJMXNO2FF37AXK:/var/lib/docker/overlay2/l/PKKMGJLCJPWNFY57NLUKCVMJBC:/var/lib/docker/overlay2/l/FIKQTNQ53CWOKGFRXBKLUBMBLD:/var/lib/docker/overlay2/l/XQEPMRHCZBNB5EVP56SU6ZFI3P:/var/lib/docker/overlay2/l/BJTP2ESCZQMRNF2VTGJJBQUV3V:/var/lib/docker/overlay2/l/WFLYIJZ2I7LQT6OWNR5RS2HAUW:/var/lib/docker/overlay2/l/IGTS4VL37ZRWDAVNFRPEUKVESA:/var/lib/docker/overlay2/l/ZGZ46Q3TAXGHSHG6ESQ2FXSNUN:/var/lib/docker/overlay2/l/AU6CATYVQXMX4LEAG2ZXXXL3LE:/var/lib/docker/overlay2/l/5UOHSDX5SG7SNWCPKSOIHNKEHJ:/var/lib/docker/overlay2/l/YGLKBNPRX4SMRAZVNSFXP775CP:/var/lib/docker/overlay2/l/DKGCRBKTARRCWMRG6O74KT2JYA:/var/lib/docker/overlay2/l/TVPIWJA7UYCGBN534BMOH5JKIZ:/var/lib/docker/overlay2/l/DXORQSSINQAALHMEXBGVBNHBI5:/var/lib/docker/overlay2/l/CPDFEGGI6TOX5OZXB473C5ENMX:/var/lib/docker/overlay2/l/2522TKFAAZK5JM5SJTFXIQDGWP:/var/lib/docker/overlay2/l/OJAMQ4FMF5SQ7BSTEH3QOHJKWT:/var/lib/docker/overlay2/l/RTA55XSMFTCHGZIWHDSDWSJPA3:/var/lib/docker/overlay2/l/3TTI2JO5YPBCYCQZW4X7Q2GKXP:/var/lib/docker/overlay2/l/AX4K2D3NVELWYZPGL224AQOJTB:/var/lib/docker/overlay2/l/PWY5HYBJ7ESPMPFCO26X6W4J3Q:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/a4a62f0d5d14ed7b08c27c1980eccbd721a133450e2c37b241bd3c43ab94e99b/diff,workdir=/var/lib/docker/overlay2/a4a62f0d5d14ed7b08c27c1980eccbd721a133450e2c37b241bd3c43ab94e99b/work,nouserxattr)
nsfs on /run/docker/netns/d8e10b8b69fc type nsfs (rw)
overlay on /var/lib/docker/overlay2/7d9fbc48528472b1ee00b75962376ce9eda4a0722804ee4adb70cc66a3daff9c/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/DOXNKZD7Y3FLS7AL6AG4MZC5LP:/var/lib/docker/overlay2/l/ENVHMJNEFC7MBQII6BR2DMCW67:/var/lib/docker/overlay2/l/3HUSGDW6CZ7GMJ536Z5JDCNG6C:/var/lib/docker/overlay2/l/3FIPLHVVYO4H7QRXCUGSFUENEY:/var/lib/docker/overlay2/l/G444UZWYIERY7XEUMCIXJUY2JU:/var/lib/docker/overlay2/l/ADJE4K3QU5CUIL4QLX65P7DV2A:/var/lib/docker/overlay2/l/K6HJEVVKYF4W37GVJIMI4WYIXM:/var/lib/docker/overlay2/l/I3EQ4U5VREA4CGHL3JC3OXRIH2:/var/lib/docker/overlay2/l/TECV7FQL3HKBU24S7HZ3TTCGFA:/var/lib/docker/overlay2/l/TLWQARASD7DQZX2LKRRHIXS36I:/var/lib/docker/overlay2/l/UXLPJDBCN67UWEOZ2FHTFCMOM6:/var/lib/docker/overlay2/l/X5Y2NTELAXQ2QFHQNMQY2XT75W:/var/lib/docker/overlay2/l/L6COJ3BUR2JO5AYEMC3BGNEI27:/var/lib/docker/overlay2/l/B5AYV2E5SX7C3CW5FBXVMLWTW4:/var/lib/docker/overlay2/l/6753I4F77QT2R4G5JAKPJYO4TV:/var/lib/docker/overlay2/l/PUNOUZSCGNDXUC3LCPFUCH6UER:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/7d9fbc48528472b1ee00b75962376ce9eda4a0722804ee4adb70cc66a3daff9c/diff,workdir=/var/lib/docker/overlay2/7d9fbc48528472b1ee00b75962376ce9eda4a0722804ee4adb70cc66a3daff9c/work,nouserxattr)
nsfs on /run/docker/netns/6d9a2406df71 type nsfs (rw)
overlay on /var/lib/docker/overlay2/4477f375f6930e844ef941749bce4567f6b02f46158d4f783b045d72d46d5d36/merged type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay2/l/YZWCXPMCZO42CZ26ZIZQOXDWPI:/var/lib/docker/overlay2/l/KE5EEP6KRKCYD5YNBZITITYYTV:/var/lib/docker/overlay2/l/NG7MLPODOWZ3OJWIE6ISOCMN7J:/var/lib/docker/overlay2/l/Z7Q2RCGYKYGE4AYPW7BNKAAC45:/var/lib/docker/overlay2/l/EJDDXEGOQSFWJSGPW2T5VT3RGP:/var/lib/docker/overlay2/l/KYZ5ZYBRO236CQIXG7STCAWNJM:/var/lib/docker/overlay2/l/K6HJEVVKYF4W37GVJIMI4WYIXM:/var/lib/docker/overlay2/l/I3EQ4U5VREA4CGHL3JC3OXRIH2:/var/lib/docker/overlay2/l/TECV7FQL3HKBU24S7HZ3TTCGFA:/var/lib/docker/overlay2/l/TLWQARASD7DQZX2LKRRHIXS36I:/var/lib/docker/overlay2/l/UXLPJDBCN67UWEOZ2FHTFCMOM6:/var/lib/docker/overlay2/l/X5Y2NTELAXQ2QFHQNMQY2XT75W:/var/lib/docker/overlay2/l/L6COJ3BUR2JO5AYEMC3BGNEI27:/var/lib/docker/overlay2/l/B5AYV2E5SX7C3CW5FBXVMLWTW4:/var/lib/docker/overlay2/l/6753I4F77QT2R4G5JAKPJYO4TV:/var/lib/docker/overlay2/l/PUNOUZSCGNDXUC3LCPFUCH6UER:/var/lib/docker/overlay2/l/A6Y2ZUX5QEG55GDPZYK6WQBVNE,upperdir=/var/lib/docker/overlay2/4477f375f6930e844ef941749bce4567f6b02f46158d4f783b045d72d46d5d36/diff,workdir=/var/lib/docker/overlay2/4477f375f6930e844ef941749bce4567f6b02f46158d4f783b045d72d46d5d36/work,nouserxattr)
nsfs on /run/docker/netns/d62869f7b5ae type nsfs (rw)
```

### What are all your local file system partition types

```shell
nvme0n1                                                                                                
├─nvme0n1p1           vfat        FAT32          736D-0CF5                               504.9M     1% /boot/efi
├─nvme0n1p2           ext4        1.0            dc069a20-5fa5-40a2-9118-430940d376f8        1G    30% /boot
└─nvme0n1p3           crypto_LUKS 2              f2a60549-c58c-412b-a57b-bc68a7a0e8b5                  
  └─nvme0n1p3_crypt   LVM2_member LVM2 001       ZSUapn-yoza-4zdC-dTD9-6Aka-Z18Z-k14gkV                
    ├─vgubuntu-root   ext4        1.0            a3a72dd0-2272-4a50-8ddf-2881330ae008    755.7G    12% /var/snap/firefox/common/host-hunspell
    │                                                                                                  /
    └─vgubuntu-swap_1 swap        1              e7b3fd82-55f6-4df7-ae18-456ce27332f5                  [SWAP]
```

### How do you use 'onedrive'

I have a pretty casual use of Onedrive, just used to store my files across devices. When trying to download it was not in active use in another system

### Steps to reproduce the behaviour

On my brand new system, ran `onedrive --sync --verbose`

### Complete Verbose Log Output

```shell
I emailed the full debug log. The normal output does not give any error, it just stops.
```

**Comment by abraunegg** [comment]
@DavidGL17 
>```
> WARNING: Your curl/libcurl version (7.81.0) has known HTTP/2 bugs that impact the use of this application.
>          Please report this to your distribution and request that they provide a newer curl version for your platform or upgrade this yourself.
>          Downgrading all application operations to use HTTP/1.1 to ensure maximum operational stability.
>          Please read https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#compatibility-with-curl for more information.
>``` 

Firstly, you are using Ubuntu thus a *broken* curl version that this client relies upon. Please read https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#compatibility-with-curl and either:

1. Manually install a newer curl version on your Ubuntu platform
2. Upgrade your Ubuntu platform to a more modern Ubuntu version
3. Upgrade your platform to Debian 12 and install 'curl' from Debian 12 Backports

If these steps are not possible, you *must* manually configure your client with the curl compatibility steps.

Secondly, having the client exit with zero reason is caused by 'curl' generating  a SIGPIPE that causes application to exit due to upstream device killing the idle TCP connection.

The only way to determine what is going on for you is to generate a HTTPS Debug Log.

#### Actions For You
1. Fix your 'curl' installation and configure the client with the required curl compatibility options
2. Once this is done, generate a HTTPS Debug Log as per: https://github.com/abraunegg/onedrive/wiki/Generate-https-debug-log-for-support

**Comment by DavidGL17** [comment]
Thank you very much for your help, it was indeed because of curl. Manually installing curl 8.9.1 resolved the issue.

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
